src/js/components/awardv2/visualizations/amounts/charts/ExceedsCurrentChart.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * ExceedsCurrentChart.jsx
+ * Created by Lizzie Salita 4/2/19
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    awardAmounts: PropTypes.object
+};
+
+export default class ExceedsCurrentChart extends React.Component {
+    render() {
+        const awardAmounts = this.props.awardAmounts;
+        const currentPercentage = Math.round((awardAmounts._combinedCurrentAwardAmounts / awardAmounts._combinedPotentialAwardAmounts) * 100);
```

Comment:
can't we use `award.currentLabelPercentage()` here? Or are you doing something different?

Reply:
I decided to move all logic for UI element width calculations out of the model and into the components. I think this makes the most sense because the calculations are fairly specific to each scenario and don't really make sense as a property of the `BaseAwardAmounts` object. I also tried to improve readability by creating a helper to convert decimal values to a percent string.